core/src/main/java/org/kohsuke/stapler/interceptor/RequirePOST.java

```diff
@@ -31,11 +33,35 @@
 @Target({METHOD,FIELD})
 @InterceptorAnnotation(RequirePOST.Processor.class)
 public @interface RequirePOST {
+
+    /**
+     * Allows customizing the error page shown when an annotated method is called with the wrong HTTP method.
+     */
+    interface ErrorHandler {
+        ForwardToView getForwardView();
```

Comment:
According to the code, it makes sense to document the `null` return here. `@CheckForNull`... you know

Reply:
Agreed, and Javadoc here is the right place to discuss the first non-null value.